apps/stable_diffusion/web/ui/txt2img_ui.py

```diff
@@ -0,0 +1,246 @@
+import os
+import sys
+import glob
+from pathlib import Path
+import gradio as gr
+from PIL import Image
+from apps.stable_diffusion.scripts import txt2img_inf
+from apps.stable_diffusion.src import (
+    prompt_examples,
+    args,
+    get_available_devices,
+)
+
+
+def resource_path(relative_path):
```

Comment:
We probably don't want to make copies of this

Reply:
Yes. will take care of this in the next clean up patch.